perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh

```diff
@@ -1,29 +1,36 @@
 #!/bin/bash
 set -e
 sudo apt-get update
+
+echo Installing git
+sudo apt-get install git
+echo Installing go-lang 1.19.5
+wget -O go_tar.tar.gz https://go.dev/dl/go1.19.5.linux-amd64.tar.gz
+sudo rm -rf /usr/local/go && tar -xzf go_tar.tar.gz && sudo mv go /usr/local
+export PATH=$PATH:/usr/local/go/bin
 echo Installing fio
 sudo apt-get install fio -y
-echo Installing gcsfuse
-GCSFUSE_VERSION=0.41.12
-curl -L -O https://github.com/GoogleCloudPlatform/gcsfuse/releases/download/v$GCSFUSE_VERSION/gcsfuse_"$GCSFUSE_VERSION"_amd64.deb
-sudo dpkg --install gcsfuse_"$GCSFUSE_VERSION"_amd64.deb
-cd "${KOKORO_ARTIFACTS_DIR}/github/gcsfuse/perfmetrics/scripts"
+
+cd "${KOKORO_ARTIFACTS_DIR}/github/gcsfuse"
+# Get the latest commitId of yesterday in the log file
+git log --before='yesterday 23:59:59' --max-count=1 --pretty=%H >> log.txt
+commitId=`cat log.txt`
+git checkout $commitId
 echo Mounting gcs bucket
 mkdir -p gcs
 LOG_FILE=log-$(date '+%Y-%m-%d').txt
-GCSFUSE_FLAGS="--implicit-dirs --max-conns-per-host 100 --disable-http2 --debug_fuse --debug_gcs --log-file $LOG_FILE --log-format \"text\" --stackdriver-export-interval=30s"
+GCSFUSE_FLAGS="--implicit-dirs --max-conns-per-host 100 --experimental-enable-storage-client-library=true --disable-http2 --debug_fuse --debug_gcs --log-file $LOG_FILE --log-format \"text\" --stackdriver-export-interval=30s"
 BUCKET_NAME=gcs-fuse-dashboard-fio
 MOUNT_POINT=gcs
 # The VM will itself exit if the gcsfuse mount fails.
-gcsfuse $GCSFUSE_FLAGS $BUCKET_NAME $MOUNT_POINT
-chmod +x run_load_test_and_fetch_metrics.sh
-./run_load_test_and_fetch_metrics.sh
+go run . $GCSFUSE_FLAGS $BUCKET_NAME $MOUNT_POINT
+chmod +x perfmetrics/scripts/run_load_test_and_fetch_metrics.sh
+./perfmetrics/scripts/run_load_test_and_fetch_metrics.sh
 # Copying gcsfuse logs to bucket
 gsutil -m cp $LOG_FILE gs://gcs-fuse-dashboard-fio/fio-gcsfuse-logs/
 
 # Deleting logs older than 10 days
-python3 utils/metrics_util.py gcs/fio-gcsfuse-logs/ 10
+python3 perfmetrics/scripts/utils/metrics_util.py gcs/fio-gcsfuse-logs/ 10
```

Comment:
nit: add a new line after logs and before starting ls tests

Reply:
fixed